js/main.js

```diff
@@ -0,0 +1,153 @@
+    data.currentWeather.description = xhr.response.data[0].weather.description;
+    $dayTripContainer.prepend(createCurrentWeather());
+  });
+  xhr.send();
+}
+
+function createCurrentWeather() {
+  var $currentWeatherRow = document.createElement('div');
+  $currentWeatherRow.className = 'row card-sky';
+
+  var $cityDateColumn = document.createElement('div');
+  $cityDateColumn.className = 'column-full';
+
+  var $headerColumn = document.createElement('div');
+  $headerColumn.className = 'column-full justify-center';
+
+  var $cityHeader = document.createElement('h3');
+  $cityHeader.className = 'mb-zero';
+  $cityHeader.textContent = data.currentWeather.destination;
+
+  var $dateColumn = document.createElement('div');
+  $dateColumn.className = 'column-full justify-center';
+
+  var $dateParagraph = document.createElement('p');
+  $dateParagraph.className = 'rm-margin';
+  $dateParagraph.textContent = data.currentWeather.date;
+
+  var $iconColumn = document.createElement('div');
+  $iconColumn.className = 'column-half justify-center';
+
+  var $icon = document.createElement('img');
+  $icon.setAttribute('src', data.currentWeather.icon);
+  $icon.setAttribute('alt', 'weather-icon');
+
+  var $tempDescriptionColumn = document.createElement('div');
+  $tempDescriptionColumn.className = 'column-half';
+
+  var $tempColumn = document.createElement('div');
+  $tempColumn.className = 'column-full';
+
+  var $tempHeader = document.createElement('h1');
+  $tempHeader.className = 'degrees rm-margin justify-center';
+  $tempHeader.textContent = data.currentWeather.temp;
+
+  var $descriptionColumn = document.createElement('div');
+  $descriptionColumn.className = 'column-full justify-center';
+
+  var $descriptionParagraph = document.createElement('p');
+  $descriptionParagraph.className = 'rm-margin';
+  $descriptionParagraph.textContent = data.currentWeather.description;
+
+  $currentWeatherRow.append($cityDateColumn);
+  $cityDateColumn.appendChild($headerColumn);
+  $headerColumn.appendChild($cityHeader);
+
+  $cityDateColumn.appendChild($dateColumn);
+  $dateColumn.appendChild($dateParagraph);
+
+  $currentWeatherRow.appendChild($iconColumn);
+  $iconColumn.appendChild($icon);
+
+  $currentWeatherRow.appendChild($tempDescriptionColumn);
+  $tempDescriptionColumn.appendChild($tempColumn);
+  $tempColumn.appendChild($tempHeader);
+
+  $tempDescriptionColumn.appendChild($descriptionColumn);
+  $descriptionColumn.appendChild($descriptionParagraph);
+
+  return $currentWeatherRow;
+}
+
+function populateBudget() {
+  $transportBudget.textContent = data.budget.transport;
+  $foodBudget.textContent = data.budget.food;
+  $activitiesBudget.textContent = data.budget.activities;
+  $souvenirsBudget.textContent = data.budget.souvenirs;
+  $reserveBudget.textContent = data.budget.reserve;
+}
+
+function handleTripSelection(event) {
+  event.preventDefault();
+  if ($tripSelect.value === "Select") {
```

Comment:
You could make the value here "home" and remove the need for a conditional here 🤓